presto-main/src/main/java/com/facebook/presto/server/StatementResource.java

```diff
@@ -164,12 +151,18 @@ public Response createQuery(
             @Context UriInfo uriInfo)
             throws InterruptedException
     {
-        assertRequest(!isNullOrEmpty(statement), "SQL statement is empty");
+        if (isNullOrEmpty(statement)) {
```

Comment:
How about we keep `ResourceUtil` around with the methods `assertRequest`, `urlEncode` and `urlDeocde`?

This class is already long, so I'd rather keep it shorter and avoid duplication.